plugins/@grouparoo/mailchimp/src/lib/export-id/exportProfile.ts

```diff
@@ -24,6 +25,7 @@ export const exportProfile: ExportProfilePluginMethod = async ({
   }
 
   return updateProfile({
+    syncOperations,
     client,
     listId,
     mailchimpId,
```

Comment:
What happens if the profile's ID changes? I'm assuming that changing the ID isn't supported, correct?

Reply:
correct.